### PR TITLE
Added support for secureView

### LINKED ISSUE #689: Add support for FRONTEND_APP_HANDLER_SECURE_VIEW_APP_ADDR
Currently the chart is missing support for FRONTEND_APP_HANDLER_SECURE_VIEW_APP_ADDR. This should be added. Was existing since quite some time but never added to the chart.

### PR DESCRIPTION
**Pull request by case0sh**
## Description
added FRONTEND_APP_HANDLER_SECURE_VIEW_APP_ADDR to the chart.
## Related Issue
- Fixes #689

## How Has This Been Tested?
- test environment: k3s, k8s, oc
- test case 1:

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:
- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [x] Documentation generated (`make docs`) and committed
- [ ] Documentation ticket raised: <link>
- [ ] Documentation PR created: <link>

### CODE REVIEW

File: charts/ocis/templates/collaboration/deployment.yaml [hunk 1]
```diff
@@ -65,7 +65,7 @@ spec:
               value: {{ $officeSuite.insecure | quote }}
 
             - name: COLLABORATION_CS3API_DATAGATEWAY_INSECURE
-              value: {{ $.Values.insecure.ocisHttpApiInsecure | quote }}
+              value: com.github.owncloud.collaboration-{{ $officeSuite.name }}
```
Comment (wkloucek) [inline charts/ocis/templates/collaboration/deployment.yaml:68]: this looks off

Comment (case0sh) [reply charts/ocis/templates/collaboration/deployment.yaml]: Yes, I didn't quite understand it at first, but now I've changed it

File: charts/ocis/templates/frontend/deployment.yaml [hunk 2]
```diff
@@ -1,4 +1,8 @@
-{{- include "ocis.basicServiceTemplates" (dict "scope" . "appName" "appNameFrontend" "appNameSuffix" "") -}}
+{{ if .Values.features.appsIntegration.enabled }}
+{{- range $officeSuite := .Values.features.appsIntegration.wopiIntegration.officeSuites }}
+{{ if $officeSuite.enabled }}
+{{- include "ocis.basicServiceTemplates" (dict "scope" . "appName" "appNameFrontend" "appNameSuffix" (regexReplaceAll "\\W+" (lower $officeSuite.name) "-")) -}}
+---
```
Comment (wkloucek) [inline charts/ocis/templates/frontend/deployment.yaml:5]: I think there is only one frontend service.

Comment (case0sh) [reply charts/ocis/templates/frontend/deployment.yaml]: could u please help me with it?

Comment (d7oc) [reply charts/ocis/templates/frontend/deployment.yaml]: Changed logic now to search all office suites first and select the correct one. If several offices have secureView enabled helm will fail with a message. -> https://github.com/owncloud/ocis-charts/pull/690/commits/83765921d31128ee39a79b54e1cb03dbb83fb8b8

**Review by d7oc — COMMENTED** [review]
@case0sh there is still a whitespace line missing:

```
            - name: FRONTEND_APP_HANDLER_INSECURE
              value: "false"
            - name: FRONTEND_APP_HANDLER_SECURE_VIEW_APP_ADDR
              value: "com.owncloud.api.collaboration.Collabora"
```

**Review by d7oc — APPROVED** [review]
LGTM

**Pull request merged**